openmc/mgxs/mgxs.py

```diff
@@ -2452,7 +2866,7 @@ def __init__(self, domain=None, domain_type=None,
         self._rxn_type = 'nu-scatter'
 
 
-class ScatterMatrixXS(MGXS):
+class ScatterMatrixXS(MatrixMGXS):
```

Comment:
I left those there as that's where all the moment and correction related code lives, which doesn't necessarily need to live in MatrixMGXS.